src/App.tsx

```diff
@@ -37,6 +37,7 @@ import createStore from 'app/store'
 import { setup as setupAnalytics } from 'app/utils/analytics'
 
 import ErrorBoundary from './ErrorBoundary'
+import { ShareDrawer } from './components/share-drawer'
```

Comment:
Looks like we are using both absolute and relative import paths in this file, let's prefer absolute here?